Adjusted SSH via VSC instructions (#722)
hirosystems/docs: content/docs/stacks/platform/guides/connect-desktop-to-cloud.mdx

1. Open VS Code on your local machine.

2. Open your user `settings.json` and add the following configuration:

> ```console

Reviewer: ```suggestion
```json
```

"remote.SSH.serverInstallPath": {
  "<your-workspace-url>": "/home/workspace/projects/",
}
```

3. Open the command palette, select "Remote-SSH: Connect to Host", then paste your `<your-workspace-url>` and hit enter.

4. A new VS Code window will open, connected to your remote server.

> 5. Click on "Open Folder" in the Explorer view and under the path `/home/workspace/projects/`, you'll be able to see all your Platform hosted projects.

Reviewer: ```suggestion
5. To view all your Platform hosted projects, click "Open Folder" in the Explorer view and navigate to `/home/workspace/projects/`.
```
